ReactiveXComponent/RabbitMq/RabbitMqHeaderConverter.cs

```diff
@@ -15,11 +15,11 @@ public static class RabbitMqHeaderConverter
 
             var dico = new Dictionary<string, object>
                            {
-                               {HeaderElement.StateMachineCode, header?.StateMachineCode ?? defaultValue},
-                               {HeaderElement.ComponentCode, header?.ComponentCode ?? defaultValue},
-                               {HeaderElement.StateCode, header?.StateCode ?? defaultValue},
-                               {HeaderElement.EventType, header?.EventCode ?? defaultValue},
-                               {HeaderElement.IncomingEventType, header?.IncomingEventType ?? defaultValue},
+                               {HeaderElement.StateMachineCode, header?.StateMachineCode},
+                               {HeaderElement.ComponentCode, header?.ComponentCode},
+                               {HeaderElement.StateCode, defaultValue},
```

Comment:
You always set the default value (-1). What if _Header_ contains a valid *StateCode* value? I think the method to fix is _RabbitMqPublisher_._CreateHeader_() as it doesn't properly set the *StateCode* property.